ballerina-tests/tests/27_return_data_bidirectional_streaming_service.bal

```diff
@@ -25,11 +24,10 @@ listener grpc:Listener ep27 = new (9117);
 }
 service "ChatFromReturn" on ep27 {
 
-    remote function chat27(stream<ChatMessage27,error?> clientStream) returns stream<string, error?>|error? {
+    remote function chat27(stream<ChatMessage27, error?> clientStream) returns stream<string, error?>|error? {
         string[] messages = [];
         int i = 0;
-        check clientStream.forEach(function(ChatMessage27 reqMsg) {
-            io:println(reqMsg);
+        checkpanic clientStream.forEach(function(ChatMessage27 reqMsg) {
```

Comment:
```suggestion
        check clientStream.forEach(function(ChatMessage27 reqMsg) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/2ef479d6607ef5aa5336331221c12c698cdd94cb